### ISSUE TITLE
Renderer Folder Structure & Watermark

### ISSUE THREAD

**Issue by SUSS-CITS** [OpenCerts/demo-opencerts-renderer#231]
Hi,

A few questions for clarification:
1. I realised that the folder structure changed. Please refer to the screenshot below. 
Currently, I'm still using the old structure. 
- I'm unable to start the program for the latest structure by using 'yarn start' command.
- Should I change it to the latest structure? 
- If I do not change it to the latest structure, is there any impact to it?  

![image](https://user-images.githubusercontent.com/49050584/70292845-50adb880-181a-11ea-9578-d91acf236ec0.png)

2. I remember before schema version 2.0 was introduced, there is watermark upon printing. This watermark feature is gone now. 
- May I know why? 
- Do we have to implement it on our own? I saw a similar issue,  https://github.com/OpenCerts/opencerts-website/issues/444

**Comment by Nebulis** [comment]
Hi,

1. We changed the demo to use a decentralized-renderer template that we built to ease the creation of renderer.

Now It's based on https://github.com/Open-Attestation/decentralized-renderer-react-template which uses https://github.com/Open-Attestation/decentralized-renderer-react-components that abstract the connection to opencerts.

I recommend you to read https://github.com/Open-Attestation/decentralized-renderer-react-template to know what's going on.

You don't need to update your current project, it's up to you. If you have any questions regarding the template we provide or if tou want to migrate to this template, I welcome you to open issues on the template repository (https://github.com/Open-Attestation/decentralized-renderer-react-template).

2. We faced many printing issues when opencerts was responsible of it. You can find more details https://github.com/OpenCerts/opencerts-website/issues/444 and are welcomed to collaborate for improvement.

Delegation to the renderer is not complete, we try to detect if the renderer is able to print by itself. If it's not then the delegation is made to opencerts. If you face any issue on this, feel free to open an issue in opencerts-website.

Delegating the printing to the renderer also means that the renderer should provide a watermark. You can use the one available in this repository, or use a custom one.

3. May I ask you:
- How you ended up following this repository as example for renderer ? 
- How you would like to get notified when we provide changes to the platform or tools you could use ?

**Comment by Nebulis** [comment]
Does that answer to your questions ?

**Comment by SUSS-CITS** [comment]
1. Noted. Since demo-opencerts-renderer is based on decentralized-renderer-react-template now, if I continue to use demo-opencerts-renderer, there won't be any issue, right? 

Just that for more information, i have to refer to decentralized-renderer-react-template?

May I know how do I run demo-opencerts-renderer now? Using yarn start command does not work now.


2. Noted. I will try and figure out on the watermark. If required on any issue about watermark, which repository do I post the issue at? 


3. I followed this repository by the updates from https://docs.opencerts.io/. As according to GovTech/SSG, the renderer is decentralized now. 

Screenshot taken from https://docs.opencerts.io/document_renderer.html. 
![image](https://user-images.githubusercontent.com/49050584/70490918-b1026a00-1b3a-11ea-8a61-a40f08cc931f.png)

In my opinion, I hope to get notified from email or through Consortium Meeting. There are always changes to OpenCerts, is hard to follow-up if I don't visit OpenCerts GitHub often. Additionally not easy for us to make changes especially when institutes rolled out to production as it might affect the current workable system. 

Thanks.

**Comment by Nebulis** [comment]
1. Yes there wont be any issue, you can even follow `decentralized-renderer-react-template` to see changes we apply on it and eventually apply them to your project if you are interested in (for instance, we will add integration testing in the coming days)
To run and develop locally we recommend to start storybook (`npm run storybook`) => all scripts available are listed on the README of both repositories

2. If facing issue developing your own renderer, can directly open an issue https://github.com/OpenCerts/opencerts-website. If you encounter an issue with the renderer we provided or have any feature request, can open in `decentralized-renderer-react-template` repository. If you are unsure, open in opencerts-website and eventually tag me, I can triage.

3. Thanks for the feedback, I will update the documentation. Do you also follow opencerts documentation update or never ?

**Comment by SUSS-CITS** [comment]
Noted for item 1 and 2. Thanks! :) 

3. Yes, i do follow opencerts documentation update too.
